content/docs/reference-react.md

```diff
@@ -66,7 +66,7 @@ See the [React.Component API Reference](/docs/react-component.html) for a list o
 
 ### `React.PureComponent`
 
-`React.PureComponent` is exactly like [`React.Component`](#reactcomponent) but implements [`shouldComponentUpdate()`](/docs/react-component.html#shouldcomponentupdate) with a shallow prop and state comparison.
+`React.PureComponent` is exactly like [`React.Component`](#reactcomponent), but implements [`shouldComponentUpdate()`](/docs/react-component.html#shouldcomponentupdate) with a shallow prop and state comparison.
```

Comment:
nit: I don't think this comma is necessary?

> implements `shouldComponentUpdate()` with a shallow prop and state comparison.

This ^ isn't an independent clause. Maybe it's an appositive?

Reply:
What am I doing with my life... 🤡